### PR TITLE
Added Pull Request Template

### PR DESCRIPTION
**Pull request by UdokaVrede**
This P.R fixes issue number #315

### CODE REVIEW

**Review by jrg94 — CHANGES_REQUESTED** [review]
This is a really cool idea! Since this is a repo mostly for articles, I don't know that some of this stuff applies. What do you think?

**Comment by UdokaVrede** [comment]
oh right I would fix this now

**Comment by jrg94** [comment]
I added some new stuff to the main README. Perhaps this could help: https://github.com/TheRenegadeCoder/sample-programs-website/blob/master/README.md

**Comment by UdokaVrede** [comment]
Alright

**Comment by UdokaVrede** [comment]
Can you give some more details pls

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Like;
+```
+ The pull request fixes issue #issue_number
+```
+## Screenshoots
+Where necessary add screenshots for your pull request
+
+## PR Type
+What kind of change does this PR introduce?
+```
```
Comment (jrg94) [inline pull_request_template.md:13]:
```suggestion

```

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+[ ] Sample program 
+[ ] Language Documentation content 
+[ ] Project Documentation content
+[ ] Other
+
+
+For a guide, on making your pull request, see [Pull_Request](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)
```
Comment (jrg94) [inline pull_request_template.md:20]:
```suggestion

```

File: pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+ The pull request fixes issue #issue_number
+```
+## Screenshoots
+Where necessary add screenshots for your pull request
+
+## PR Type
+What kind of change does this PR introduce?
+```
+[ ] Sample program 
+[ ] Language Documentation content
```
Comment (jrg94) [inline pull_request_template.md:15]:
```suggestion
- [ ] Language Documentation
```

File: pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+```
+## Screenshoots
+Where necessary add screenshots for your pull request
+
+## PR Type
+What kind of change does this PR introduce?
+```
+[ ] Sample program 
+[ ] Language Documentation content 
+[ ] Project Documentation content
```
Comment (jrg94) [inline pull_request_template.md:16]:
```suggestion
- [ ] Project Documentation
```

File: pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+```
+ The pull request fixes issue #issue_number
+```
+## Screenshoots
+Where necessary add screenshots for your pull request
+
+## PR Type
+What kind of change does this PR introduce?
+```
+[ ] Sample program
```
Comment (jrg94) [inline pull_request_template.md:14]:
```suggestion
- [ ] Sample Program Documentation
```

**Review by jrg94 — CHANGES_REQUESTED** [review]
Looking better! I think we can more or less just use this check list. Everything else can probably be removed. 👍

**Review by UdokaVrede — COMMENTED** [review]
This looks good, thanks

**Pull request merged**